apps/experiments/migrations/0072_load_openai_voices.py

```diff
@@ -0,0 +1,41 @@
+# Generated by Django 4.2.7 on 2024-03-25 08:50
+
+from django.db import migrations
+import os
+import json
+
+def load_synthetic_voices(apps, schema_editor):
```

Comment:
would be great to abstract this and the other 2 methods (in 0032 and 0044) to a utility function